README.md

```diff
@@ -1,3 +1,15 @@
 # Signadot Examples
 
 This repository contains examples for Signadot users.
+
+# Setup Instructions for Running Examples
+
+1. Ensure you have [Docker](https://www.docker.com/), [minikube](https://minikube.sigs.k8s.io/docs/), [helm](https://helm.sh/), and [kubectl](kubectl apply -f https://raw.githubusercontent.com/signadot/hotrod/main/k8s/all-in-one/demo.yaml) installed.
+2. Run [`minikube start`](https://minikube.sigs.k8s.io/docs/start/) to create a Kubernetes environment locally.
```

Comment:
There's an instruction to create the namespace `hotrod` - `kubectl create ns hotrod`